src/coffee/services/product-projections.coffee

```diff
@@ -250,7 +250,6 @@ class ProductProjectionService extends BaseService
   #   .search()
   search: ->
     @_currentEndpoint = '/product-projections/search'
-    @fetch()
```

Comment:
It's a matter of documentation ;)

See https://github.com/sphereio/sphere-node-sdk/issues/75